src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/DefaultTagHelperServiceTestBase.cs

```diff
@@ -141,6 +172,12 @@ internal static RazorCodeDocument CreateCodeDocument(string text, params TagHelp
             return CreateCodeDocument(text, "test.cshtml", tagHelpers);
         }
 
+        protected CompletionCapability CompletionCapability { get; } = new CompletionCapability {
```

Comment:
This seems too specific for a TestBase, I'd just add this to the test impl type.